packages/client/components/ScopingSearchInput.tsx

```diff
@@ -36,11 +38,14 @@ interface Props {
   queryString: string
   meetingId: string
   linkedRecordName: string
+  service: IntegrationProviderServiceEnum | 'PARABOL'
```

Comment:
-1 We should use `TaskServiceEnum` from `__generated__/SendClientSegmentEventMutation.graphql` here instead.

Reply:
ah totally make sense! Fixed